doc/flakes/design.md

```diff
@@ -2,92 +2,80 @@
 
 ## Goals
 
-* To provide Nix repositories with an easy and standard way to
-  reference other Nix repositories.
+* Standard and easy dependency handling in Nix
```

Comment:
This is ambiguous because it doesn't explain what is meant with dependency handling. I mean, Nix has always done "dependency handling" between packages, it just didn't have dependency handling for Nix repos.